src/pages/support-page/index.vue

```diff
@@ -0,0 +1,226 @@
+                type="textarea"
+                rows="7"
+                class="input-question__scrollbar"
+                resize="none"
+              />
+            </el-form-item>
+          </div>
+          <el-form-item>
+            <el-checkbox v-model="ruleForm.agreePrivacyPolicy">
+              I agree with the <el-link type="primary" @click="navigateToPrivacyPolicy">Privaci Poilicy</el-link>
+            </el-checkbox>
+          </el-form-item>
+          <el-form-item>
+            <el-button
+              class="support-page__send-button"
+              type="primary"
+              :disabled="!ruleForm.agreePrivacyPolicy"
+              @click="submitForm(ruleFormRef)"
+            >
+              Send
+            </el-button>
+          </el-form-item>
+        </el-form>
+      </div>
+    </div>
+    <div class="support-contacts">
+      <h3 class="support-contacts__title">Contact with us</h3>
+      <div class="support-contacts__list">
+        <div class="support-contacts__item">
+          <div class="support-contacts__item-main">
+            <img class="support-contacts__item-image" src="@@/assets/images/Icon-VR.png" alt="support telegram bot" />
+            <div class="support-contacts__item-caption">
+              <h4 class="support-contacts__item-title">Support Telegram Bot</h4>
+              <span class="support-contacts__item-description">Write here if you have any problems</span>
+            </div>
+          </div>
+          <div class="support-contacts__button"><h4 class="support-contacts__button-text">GO</h4></div>
+        </div>
+        <div class="support-contacts__item">
+          <div class="support-contacts__item-main">
+            <img
+              class="support-contacts__item-image"
+              src="@@/assets/images/Icon-EmailVerification.png"
+              alt="support telegram chat"
+            />
+            <div class="support-contacts__item-caption">
+              <h4 class="support-contacts__item-title">News Telegram Channel</h4>
+              <span class="support-contacts__item-description">Updates, news and opening levels</span>
+            </div>
+          </div>
+          <div class="support-contacts__button"><h4 class="support-contacts__button-text">GO</h4></div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts" setup>
+import { reactive, ref } from 'vue'
+import { useRouter } from 'vue-router'
+import type { FormInstance, FormRules } from 'element-plus'
+
+const ruleFormRef = ref<FormInstance>()
+const router = useRouter()
+
+const navigateToPrivacyPolicy = (event: MouseEvent) => {
+  event.preventDefault()
+
+  router.push(router.currentRoute.value.path)
+}
+
+const ruleForm = reactive({
+  name: '',
+  email: '',
+  question: '',
+  agreePrivacyPolicy: false,
+})
+
+const rules = reactive<FormRules>({
+  name: [{ required: true, trigger: 'blur' }],
+  email: [{ required: true, trigger: 'blur' }],
```

Comment:
```suggestion
  email: [{ required: true, trigger: 'blur', type: 'email'}],
```

Но это не точно. Нужно прокинуть email для дополнительной валидации